bedrock/firefox/views.py

```diff
@@ -584,11 +585,9 @@ def download_thanks(request):
         elif experience == 'auf-deiner-seite':
             template = 'firefox/new/berlin/scene2-auf-deiner-seite.html'
         else:
-            template = 'firefox/new/scene2.html'
+            template = 'firefox/new/scene2_email.html'
```

Comment:
Hm though if we change that condition it could break a bunch of the "non-English" tests that use 'de' as the test locale. I don't know if those tests will return true for lang_file_is_active... maybe they do? It's true in dev mode but I think they might fail when testing prod...

Maybe we should change all those tests to use a locale less complicated than German, at least as long as we have all these special German pages.

Reply:
@craigcook oh great catch! A second pair of eyes here is always good 👀 👍 

We can mock the lang file active status in the tests should we need, but it should always be true for Dev.